projects/components/src/data-exporter/data-exporter.component.html

```diff
@@ -77,7 +79,7 @@ <h3 class="modal-title">{{ dialogHeader | lazyString }}</h3>
             </ul>
             <div [ngStyle]="{ visibility: shouldShowBubbles ? 'visible' : 'hidden' }" class="column-container">
                 <span class="label column-label" *ngFor="let column of selectedColumns">
-                    {{ this.friendlyFieldsControl.value ? column.displayName : column.fieldName }}
+                    {{ this.friendlyFieldsControl.value ? column.displayName || column.fieldName : column.fieldName }}
```

Comment:
The point of my last request was so we wouldn't have to duplicate the defaulting here, you should be using the function that gets the display name.